SDK/AppCenter/Microsoft.AppCenter.iOS.Bindings/ApiDefinition.cs

```diff
@@ -204,6 +204,10 @@ interface MSAppCenter
         [Export("setLogLevel:")]
         void SetLogLevel(MSLogLevel logLevel);
 
+        [Static]
```

Comment:
Looks like a comment with original ObjC signature is missing (we have it for other bindings):
`// + (void)setUserId:(NSString *)userId;`